### DISCUSSION TITLE
想使用西门子博图远程调试，出现问题，子网可以ping通，可访问节点找不到

### DISCUSSION THREAD

**Discussion by langsisi** [EasyTier/EasyTier#506] [Q&A]
![image](https://github.com/user-attachments/assets/1b63134e-9599-49ef-af10-571a4f906ae3)
可以转在线，可以ping通，说明网络是OK的
![image](https://github.com/user-attachments/assets/ec979516-666b-49fd-b69b-020d4c8bb58f)
西门子可访问节点里面没有找到设备
![image](https://github.com/user-attachments/assets/30e8dfbb-1dda-443c-8c70-d8eecd8c136c)
用kuka的workvisual可以找到kuka机器人，再次说明网络没有问题
查找西门子官方资料，有如下信息，本人小白，不懂什么意思：
西门子博图使用DCP/LLDP 收集信息，搜索西门子设备。路由器如果不支持转发LLDP数据包，会影响最终读取的到的设备信息。建议使用有线网卡和路由器、交换机，不要使用无线路由器

**Comment by KKRainbow** [comment]
LLDP 是 L2 的协议，ET 目前只支持 TUN 设备，也就是 L3 协议的转发

**Comment by lurenjiamax** [comment]
可以用三层的组网软件试试, 比如zerotier
